dLux/apertures.py

```diff
@@ -0,0 +1,2641 @@
+import equinox as eqx
+import matplotlib as mpl 
+import matplotlib.pyplot as plt
+import jax.numpy as np
+import jax 
+import dLux
+import abc
+import functools
+from typing import TypeVar 
+
```

Comment:
So we want to import specific functions or classes from packages, not the whole package (check the CONTRIBUTING.md file)

Reply:
How does this work with `import jax.numpy as np`. Are we switching to `from jax import numpy as np`?